### PR TITLE
Don't copy libraries that are already in prime.

### PR DESCRIPTION
**Pull request by kyrofa**
Currently snapcraft is fooled by relative rpaths and copies linked libraries, even if they've already been primed. This PR fixes LP: [#1570895](https://bugs.launchpad.net/snapcraft/+bug/1570895) and stops that by checking each library path to ensure it hasn't already been primed before doing so.

### CODE REVIEW

File: integration_tests/test_rpath.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2015 Canonical Ltd
```
Comment (sergiusens) [inline integration_tests/test_rpath.py]: 2016 is the current year. Stop living in the past :tongue:

Comment (kyrofa) [inline integration_tests/test_rpath.py]: This was terrible at the very beginning of the year. I kept feeling like it was 2015, so it was fine to see it. Then in, say, February, I started catching files that contained 2015 and putting 2016 in there. Now we're like halfway through the year, and I've apparently started assuming that all the copyrights say 2016 already :stuck_out_tongue: .

Comment (kyrofa) [inline integration_tests/test_rpath.py]: Fixed.

**Comment by sergiusens** [comment]
ok to test

**Comment by kyrofa** [comment]
retest this please.

**Pull request merged**